nix/status-go/library/default.nix

```diff
@@ -18,6 +18,10 @@ in buildGoPackageIntegrationTest {
   # TODO: try removing when go is upgraded to 1.22
   GODEBUG = "netdns=cgo+2";
 
+  # since go 1.21 status-go compiled binary needs access to go compiler at buildtime
+  # FIXME: remove this when go 1.23 or later versions fix this madness
```

Comment:
```suggestion
  # Since go 1.21 status-go compiled library includes references to cgo runtime.
  # FIXME: remove this when go 1.23 or later versions fix this madness
```